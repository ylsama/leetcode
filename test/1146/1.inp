["SnapshotArray","set","snap","set","snap","get","get","set","get","set"]
[[2],[0,4],[],[1,13],[],[1,1],[1,0],[1,3],[1,0],[0,5]]