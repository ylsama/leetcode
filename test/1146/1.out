[null,null,0,null,1,13,13,null,13,null]